functions/brinks/src/lib/isEventProcessed.js

```diff
@@ -0,0 +1,16 @@
+// determines if stripe event has already been processed
+import debug from 'debug';
+import { Firestore } from '@google-cloud/firestore';
+import { dataSources } from '@thatconference/api';
+
+const dlog = debug('that:api:functions:brinks:isEventProcessed');
+const firestore = new Firestore();
```

Comment:
and again.

Reply:
Not a connection at this point, only a reference to the function. We could consolidate any such references to a request reference if you feel it will be better. It is something I have not done so don't have a good opinion on. Being a function reference I don't see it being 'expensive'.

Reply:
Refactored in brinks: ac099c2